powerline/bindings/lemonbar/powerline-lemonbar-multimonitor.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import sys
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.monotonic import monotonic
+from powerline.lib.dict import mergedicts
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(description='Powerline BAR bindings.')
+    parser.add_argument(
+        '--i3', action='store_true',
+        help='Subscribe for i3 events.'
+    )
+    args = parser.parse_args()
+
+    themes = {
+        'LVDS1': 'default',
+        'HDMI1': 'hdmi',
+        'VGA1': 'hdmi'
+    }
+
+    active_screens = [
+        dict(zip(['name', 'width', 'height', 'x', 'y'], values))
+        for values in re.findall(
+            '^(\w+) connected (\d+)x(\d+)\+(\d+)\+(\d+)',
```

Comment:
I would actually suggest using named capturing groups and [groupdict](https://docs.python.org/2.6/library/re.html?highlight=re#re.MatchObject.groupdict) in place of dict+zip.